src/diffpy/utils/parsers/data_loader.py

```diff
@@ -16,7 +16,7 @@
 import numpy
 
 
-def loadData(filename, minrows=10, headers=False, hdel="=", hignore=None, **kwargs):
+def load_data(filename, minrows=10, headers=False, hdel="=", hignore=None, **kwargs):
```

Comment:
we do not want this change (please see the discussion on #163 ).  Please revert.